docs/modular-resources/key-concepts.md

```diff
@@ -36,24 +36,21 @@ A module provides definitions for one or more pairs of [APIs](#valid-apis-to-imp
 
 When the module initializes, it registers those pairs on your robot, making the functionality defined by that pair available for use.
 
-You can [upload your own modules to the Viam registry](/modular-resources/upload/) or can [add existing modules from the Registry](/modular-resources/configure/).
-
-See [Creating a custom module](/modular-resources/create/) for more information.
-
 ## Resources
 
 A resource is a [component](/components/) or [service](/services/).
 Each component or service is typed by a proto API, such as the [component proto definitions](https://github.com/viamrobotics/api/tree/main/proto/viam/component).
 
-Any resource on your robot needs to implement either one of these [existing Viam APIs](#valid-apis-to-implement-in-your-model), or a custom interface.
+Any resource on your robot needs to implement either one of the [existing Viam APIs](#valid-apis-to-implement-in-your-model), or a [custom interface](/modular-resources/advanced/#new-api-subtypes).
 
-A _modular resource_ is a resource that is provided by a [module](#modules), and not built-in to the RDK.
-A modular resource runs in the module process. This differs from built-in resources, which run as part of `viam-server`.
+A _modular resource_ is a resource that is provided by a [module](#modules), and not built into the RDK.
+A modular resource runs in the module process.
+This differs from built-in resources, which run as part of `viam-server`.
 
 ## Models
 
 A _model_ describes a specific implementation of a [resource](#resources) that implements (speaks) its [API](/program/apis/).
-Models allow you to control different instances of a resource with a consistent interface, even if the underlying implementation differs.
+Models allow you to control hardware or software of a similar category, such as motors, with a consistent set of methods as an interface, even if the underlying implementation differs.
```

Comment:
"with a consistent set of methods as an interface" sounds like it's missing a direct object. Is there a way to split this sentence to help clarify what we mean?

Reply:
```suggestion
Models allow you to use the same set of methods to control hardware or software of a similar category, even if the underlying implementation differs.
```
Is this better?

Reply:
Perfect!